src/1-raw-data/mailinglists/mailman_data_test.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestCreateMailManURL(t *testing.T) {
+	*mailingListURL = "https://en.wikipedia.org/wiki/Susan_La_Flesche_Picotte"
+	filename := "susan_la_flesche_picotte.mbox.gz"
+	startDate := "1865-06-17"
+	endDate := "1915-09-18"
+	expected := *mailingListURL + "export/python-dev@python.org-" + filename + "?start=" + startDate + "&end=" + endDate
+
+	actual := createMMURL(filename, startDate, endDate)
+
+	if strings.Compare(expected, actual) != 0 {
+		t.Errorf("CreateMMURL response does not match.\n got: %v\nexpected: %v", actual, expected)
```

Comment:
I'd take out the newline. Common structure is: Message; got:, want:

Reply:
Newline helps my eyes jump to it in the response